commands/exportcommands.py

```
@@ -120,18 +119,36 @@ def _export_commands(self, staging_dir):
 
-                print(f"    {filename}")
+                print(f"   • {filename}")
                 self.commands_count += 1
 
-        print(f"Success Exported {self.commands_count} commands")
+        print(f"✅ Exported {self.commands_count} commands")
+
+    def _copy_hooks_manual(self, hooks_dir, target_dir):
+        """Windows fallback - manual directory copy with filtering"""
+        import shutil
+        for root, dirs, files in os.walk(hooks_dir):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)